page_analyzer/templates/url.html

```diff
@@ -42,7 +42,18 @@ <h2 class="mt-5 mb-3">Проверки</h2>
             <th>Дата создания</th>
           </tr>
         </thead>
-        <tbody></tbody>
+        {% if checks %} {% for check in checks %}
```

Comment:
I have a feeling that `if` check here is not necessary - nothing will be rendered if `checks` is empty

Reply:
there were None values in h1, status_code, description and title columns, so I use this "if" to avoid None values.
Now I add default values  to table scheme and there's no need for "if

Reply:
no, this is not what I meant :^)
let's take a look at two imaginary pieces of code and see if they have any behavioral differences

```
checks = []
if checks:
    for check in checks:
        print(check)
```

```
checks = []
for check in checks:
    print(check)
```

will they behave differently?

Reply:
I see. thanks :)